Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+GLPK = "60bf3e95-4087-53dc-ae20-288a0d20c6a6"
```

Comment:
This package suggests HiGHS as the default solver, like the JuMP docs. However we always provide an optimizer argument so that users can change this setting. 
Can you remove the GLPK dependency?